pkg/utils/logger/logger_test.go

```diff
@@ -0,0 +1,121 @@
+package logger_test
+
+import (
+	"encoding/json"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/stretchr/testify/require"
+	"github.com/trisacrypto/directory/pkg/utils/logger"
+)
+
+type testWriter struct {
+	lastLog map[string]interface{}
+	levels  map[zerolog.Level]uint16
+}
+
+func (w *testWriter) WriteLevel(level zerolog.Level, p []byte) (n int, err error) {
+	if w.levels == nil {
+		w.levels = make(map[zerolog.Level]uint16)
+	}
+	w.levels[level]++
+	return w.Write(p)
+}
+
+func (w *testWriter) Write(p []byte) (n int, err error) {
+	if err = json.Unmarshal(p, &w.lastLog); err != nil {
+		return 0, err
+	}
+	return len(p), nil
+}
+
+func TestSeverityHook(t *testing.T) {
```

Comment:
These tests didn't exist before so I figured I'd try to bump our coverage a bit.